nix/pkgs/libspdk/default.nix

```diff
@@ -133,6 +133,7 @@ let
       install -v build/lib/pkgconfig/*.pc        $out/lib/pkgconfig/
       install -v dpdk/build/lib/*.a              $out/lib/
       install -v dpdk/build/lib/pkgconfig/*.pc   $out/lib/pkgconfig/
+    '' + lib.optionalString targetPlatform.isx86_64 ''
       install -v intel-ipsec-mb/lib/*.a          $out/lib/
```

Comment:
q: Is there an alternative to ipsec library in arm64? Just thinking if its used in only x64,  what is the corresponding in arm64.

Reply:
AFAIK, intel-ipsec-mb only supports intel x64 CPUs, because it is implemented in[ intel individual acceleration instructions](https://github.com/intel/intel-ipsec-mb/tree/main/lib). And for arm64 there is [one armv8 crypto lib ](https://doc.dpdk.org/guides/cryptodevs/armv8.html )in DPDK which haven been integrated in Mayastor yet.

Reply:
And how Mayastor use intel-ipsec-mb? Because I [remove the crypto lib ](https://github.com/mayadata-io/spdk-rs/pull/9/files#diff-d0d98998092552a1d3259338c2c71e118a5b8343dd4703c0c7f552ada7f9cb42R138)it still can be compiled and run on arm64.